modules/olm-operatorgroups-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="olm-operatorgroups-limitations"]
+= Limitations for multi-tenant Operator management
+
+{product-title} provides limited support for simultaneously installing different variations of an Operator on a cluster. Operators are control plane extensions. All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multi-tenant environment also have to share Operators.
+
+Operator Lifecycle Manager (OLM) installs Operators multiple times in different namespaces and one of the constraints being that the Operator’s API versions need to be the same.
```

Comment:
I think this sentence needs split into two because the second half reads oddly (imo). Maybe something like 'The Operator Lifecycle Manager (OLM) installs Operators multiple times in different namespaces. One constraint of this is that the Operator's API versions must be the same." 

Another way might be better? Just a suggestion. 

(swap need to/ must based on https://docs.engineering.redhat.com/display/CCSENABLE/Phrasal+verbs).